### PR TITLE
add patch for zlib CVE-2022-37434

### PR DESCRIPTION
**Pull request by rawlingsj**
Signed-off-by: James Rawlings <jrawlings@chainguard.dev>

### CODE REVIEW

**Comment by rawlingsj** [comment]
In draft as still verifying

File: zlib.yaml [hunk 1]
```diff
@@ -13,6 +13,12 @@ package:
   dependencies:
     runtime:
 
+secfixes:
+  1.2.12-r3:
+    - CVE-2022-37434
+  1.2.11-r4:
```
Comment (rawlingsj) [inline zlib.yaml:19]: should this instead be the first version of wolfi zlib?  i.e. `1.2.12-r2`

Comment (rawlingsj) [reply zlib.yaml:19]: @kaniini just checking ^

Comment (kaniini) [reply zlib.yaml:19]: 1.2.12-r0

**Pull request closed without merging**